app/src/main/java/com/owncloud/android/operations/DownloadFileOperation.java

```diff
@@ -105,11 +107,11 @@ public String getSavePath() {
     }
 
     public String getTmpPath() {
-        return FileStorageUtils.getTemporalPath(user.getAccountName()) + file.getRemotePath();
+        return FileStorageUtils.getInternalTemporalPath(user.getAccountName(), context) + file.getRemotePath();
```

Comment:
```suggestion
        return getTmpFolder() + file.getRemotePath();
```
Also most of these methods look like they can be private